app/services/nomis/models/offender.rb

```diff
@@ -28,14 +28,36 @@ class Offender
       attribute :reception_date, :date
       attribute :marital_status, :string
       attribute :main_offence, :string
-      attribute :release_date, :date
-      attribute :sentence_date, :date
       attribute :tier, :string
       attribute :case_allocation, :string
       attribute :omicable, :boolean
       attribute :allocated_pom_name, :string
+      attribute :release_date, :date
+      attribute :sentence_date, :date
       attribute :parole_eligibility_date, :date
-      attribute :has_indeterminate_release_date
+      attribute :home_detention_curfew_eligibility_date, :date
+      attribute :tariff_date, :date
+
+      def sentence_detail=(sentence_detail)
+        self.release_date = sentence_detail.release_date
+        self.sentence_date = sentence_detail.sentence_date
```

Comment:
A tiny question: can we rename this to be sentence_start_date?  Sentence date could potentially cause some confusion if the context/datapoint isn't known.

Reply:
Good idea.